### ISSUE TITLE
Only log files with unused dependencies

### ISSUE THREAD

**Issue by pdehaan** [mehdishojaei/grunt-amdcheck#7]
I'm trying to use this grunt plugin but cannot find the right config settings to remove some of the verbose output.
Currently if I run `grunt amdcheck` in our project, I get about 317 lines of results:
```
app/scripts/head/boot.js (no module)
app/scripts/head/start.js (no module)
app/scripts/head/startup-styles.js (1 module)
Unused paths: lib/environment
app/scripts/lib/able.js (1 module)
app/scripts/lib/app-start.js (1 module)
app/scripts/lib/assertion.js (1 module)
app/scripts/lib/auth-errors.js (1 module)
...
```

I'm trying to find the right config combination to only show results for files with unused paths:
```
app/scripts/head/startup-styles.js (1 module)
Unused paths: lib/environment
```

Currently I'm hacking using something like this (which reduces my output to 132 lines):
```sh
$ grunt amdcheck | grep "Unused paths:" -B 1 | grep -v '\-\-'
```

```
app/scripts/head/startup-styles.js (1 module)
Unused paths: lib/environment
app/scripts/lib/channels/fx-desktop.js (1 module)
Unused paths: lib/auth-errors
app/scripts/lib/config-loader.js (1 module)
Unused paths: underscore
app/scripts/lib/fxa-client.js (1 module)
Unused paths: underscore, lib/xhr
app/scripts/lib/mailcheck.js (1 module)
Unused paths: mailcheck
app/scripts/lib/oauth-client.js (1 module)
Unused paths: lib/promise, lib/session
...
```

---

My current config looks like this:
```js
module.exports = function (grunt) {
  'use strict';

  grunt.config('amdcheck', {
    app: {
      options: {
        removeUnusedDependencies: false,
        logFilePath: true
        // logUnusedDependencyPaths: true
      },
      files: [{
        expand: true,
        cwd: 'app/',
        src: [
          '**/*.js',
          '!bower_components/**'
        ]
      }]
    }
  });
}
```

**Comment by mehdishojaei** [comment]
@pdehaan Thanks for your issue report. I'm working on now...

**Comment by mehdishojaei** [comment]
@pdehaan I removed the option `logFilePath`, and only file path of the files with at least one unused dependency will be logged. So the output must not be verbose. Version upgraded to `1.3.0`.

**Comment by pdehaan** [comment]
Awesome, thanks!
New output looks great.

**Comment by pdehaan** [comment]
Closing. Fixed by 4986018baac49428740ee7ed5074512a2d6b4713
